hermes-common/src/main/java/pl/allegro/tech/hermes/common/ssl/SslContextFactory.java

```diff
@@ -1,8 +1,11 @@
 package pl.allegro.tech.hermes.common.ssl;
 
 import javax.net.ssl.SSLContext;
+import javax.net.ssl.TrustManager;
 
 public interface SslContextFactory {
 
     SSLContext create();
+
+    TrustManager[] getTrustManagers();
```

Comment:
As we talked already, we could change this interface, so it would have a single method:
`SSLContextHolder create()` or something similar... :)
and `SSLContextHolder` would hold both the `TrustManagers[]` and `SSLContext` instances. This way we explicitly say that these object always stick together and are created at the same time by the factory.